tests/e2e/democracy.go

```diff
@@ -7,15 +7,14 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
 	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	ibctestingutils "github.com/cosmos/interchain-security/testutil/ibc_testing"
```

Comment:
Maybe alias as `icstestingutils`, `icsibctestingutils`. `icsutilsibc`

Can't think of anything better but think that differentiating from `ibctesting` is useful

Reply:
Agreed, there's not a super intuitive name that I can think of. But I've at least changed the alias to `icstestingutils`